### DISCUSSION TITLE
Chat Completion error after submitting prompt

### DISCUSSION THREAD

**Discussion by jack-vinitsky** [microsoft/sample-app-aoai-chatGPT#877] [Q&A]
Greetings,

I just the deployed the solution via Azure AI Studio, and I am getting the following error after submitting a prompt:

There was an error generating a response. Chat history can't be saved at this time. Error code: 400 - {'error': {'code': 'OperationNotSupported', 'message': 'The chatCompletion operation does not work with the specified model, . Please choose different model and try again. You can learn more about which models can be used with each operation here: https://go.microsoft.com/fwlink/?linkid=2197993.'}}

I have confirmed that I have environment variable values for: AZURE_OPENAI_MODEL & AZURE_OPENAI_MODEL_NAME

**Comment by jack-vinitsky** [comment]
It turned out I had the wrong name specified for AZURE_OPENAI_MODEL.  Putting in the correct name solved the issue.

**Reply by 2gad** [reply]
can confirm -- a similar issue when going to get-4-mini; 
deployed with: gpt4omini
got the same error
updated to: gpt-4o-mini

works great
